### PR TITLE
Hook Service spec for Hook App with github app and Permissions screenshot

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
- This PR adds the hook service spec for Hook App created with GitHub App. (Missed to add this earlier)
- Also adding the screenshot of Github App Permissions. (Need this to link in Wiki Page)

### CODE REVIEW

File: config/prow/cluster/hook_github_app_service.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (mkumatag) [inline config/prow/cluster/hook_github_app_service.yaml:1]: in most of the place we just use the naming convention used for the file name is like `<name>_<type>.yaml` e.g: hook_service.yaml, hence can we rename this file as `hook2_service.yaml`

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Review by mkumatag — CHANGES_REQUESTED** [review]
and I see this PowerCloudCI App Permissions.png is also attached, I'm wondering why here in the root dir?

**Comment by Rajalakshmi-Girish** [comment]
> 
> 
> and I see this PowerCloudCI App Permissions.png is also attached, I'm wondering why here in the root dir?

I saw .png images in the logos folder but that doesn't seem to be appropriate for this screenshot.
Please let me know if a folder should be created for this.

**Comment by mkumatag** [comment]
> > and I see this PowerCloudCI App Permissions.png is also attached, I'm wondering why here in the root dir?
> 
> I saw .png images in the logos folder but that doesn't seem to be appropriate for this screenshot.
> Please let me know if a folder should be created for this.

is this for wiki page or something? if yes - lets create some folder with name images or so and push it there.

**Comment by Rajalakshmi-Girish** [comment]
> 
> 
> > > and I see this PowerCloudCI App Permissions.png is also attached, I'm wondering why here in the root dir?
> > 
> > 
> > I saw .png images in the logos folder but that doesn't seem to be appropriate for this screenshot.
> > Please let me know if a folder should be created for this.
> 
> is this for wiki page or something? if yes - lets create some folder with name images or so and push it there.

Yes this is for wiki page.
The folder `images` already has the Dockerfiles for the container images.
@mkumatag how about the folder name as `screenshots`.

**Comment by mkumatag** [comment]
> Yes this is for wiki page.
> The folder `images` already has the Dockerfiles for the container images.
> @mkumatag how about the folder name as `screenshots`.

sure

File: config/prow/cluster/hook_github_app_service.yaml (hunk 1 above)
Comment (Rajalakshmi-Girish) [reply config/prow/cluster/hook_github_app_service.yaml]: Done.

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Pull request merged**